specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -15,36 +15,25 @@
 - [Alternatives](#alternatives)
 
 ## Summary
-
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
+At the moment customers can deploy and manage jobs through the VDK CLI tool or through the API's.
+We want to extend on this and create a web frontend for the VDK.
+The frontend will let customers, create,delete deploy and generally managed nad browse their jobs.
+Frontend is written in an extensible format to easily allow other teams to build extensions.
+To achieve this extensibility the frontend will be built in angular.
 
 ## Glossary
-<!--
-Optional section which defines terms and abbreviations used in the rest of the document.
--->
+data-pipelines: This is the npm project of the bootstrapping codebase it contains all the vdk specific frontend code. for more details please see [README.md](/projects/frontend/data-pipelines/README.md)
+shared-components, @vdk/shared: This is a set of components that are used within the data-pipelines project but are generic and could be re-used by other projects [README.md](/projects/frontend/data-pipelines/README.md)
 
 ## Motivation
-<!--
-It tells **why** do we need X?
-Describe why the change is important and the benefits to users.
-Explain the user problem that need to be solved.
--->
+We want to improve the usability of VDK. With a UI customers will be able to use VDK much quicker and much more proficiently because UIs are easier to understand and become familiar with than CLI tools or APIs.
```

Comment:
I think "much quicker" or "much more proficient" depends on the user audience. For example the GUI could be easier for a novice user. Also, GUI is usually utilised by more people than CLI. Yet some people may find it rather quicker to configure and use a CLI. 
So my suggestion is to rather elaborate on the broader audience and user friendliness, instead of generalising the speed of usage.